forkmanager/fork.go

```diff
@@ -0,0 +1,43 @@
+package forkmanager
+
+type ForkHandlerName string
+
+type ForkName string
+
+type ForkInfo struct {
+	Name            ForkName
+	FromBlockNumber uint64
+}
+
+type Fork struct {
+	Name            ForkName
+	FromBlockNumber uint64
+	IsActive        bool
```

Comment:
Why do I need this? Isn't it active as soon as it reaches predefined block number?

Reply:
Because fork can be registered but not activated. I did not want to use *uint64 for `FromBlockNumber` so I use two fields for same effect (but its more memory access friendly because its not indirect pointer access).

Remember that fork manager can be used even for dynamic forks from smart contract